r/tools/nixlibs.R

```diff
@@ -99,8 +99,8 @@ download_binary <- function(lib) {
 # * Some other string: a "distro-version" that corresponds to a binary that is
 #   available, to override what this function may discover by default.
 #   Possible values are:
-#    * "centos-7" (gcc 8 (devtoolset), openssl 1, glib 2.17)
-#    * "ubuntu-18.04" (gcc 8, openssl 1, glib 2.27)
+#    * "centos-7" (gcc 8 (devtoolset), _GLIBCXX_USE_CXX11_ABI=0, openssl 1.0)
```

Comment:
I think renaming the other centos7 binary would communicate the difference between the two better (without having to look in nixlibs.R)?
```suggestion
#    * "centos-7-openssl-1.0" (gcc 8 (devtoolset), _GLIBCXX_USE_CXX11_ABI=0, openssl 1.0)
```

Reply:
I kept the `centos-7` name for backward compatibility. (I thought that some users may specify `centos-7` explicitly.)

I'll rename them as the followings:

* `centos-7` -> `openssl-1.0`
* `centos-7-openssl-1.1` -> `openssl-1.1`
* `ubuntu-22.04` -> `openssl-3.0`